mne/beamformer/_lcmv.py

```diff
@@ -168,7 +168,36 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
     Returns
     -------
     filters | dict
-        Beamformer weights.
+        Dictionary containing filter weights from LCMV beamformer.
+
+        Parameters
+        ----------
```

Comment:
@dengemann the Docstring check complains about specifying Parameters twice in the same function - any settings of what to use instead?